Makefile

```diff
@@ -4,6 +4,11 @@ APP_NAME := finch
 FINCH_WPS_URL := http://localhost:5000
```

Comment:
Create PR to replace FINCH_WPS_URL by WPS_URL in the code and notebooks.

Reply:
I'll handle in this PR together with  TravisCI and possibly RtD.